tardis/montecarlo/base.py

```diff
@@ -220,3 +222,44 @@ def calculate_f_nu(self, frequency):
 
     def calculate_f_lambda(self, wavelength):
         pass
+
+    def property_to_hdf(self, property):
+        output_value = getattr(self, property)
+        if np.isscalar(property):
+            output_value = [property]
+
+        return pd.DataFrame(output_value)
+
+    def to_hdf(self, path_or_buf, path='', close_hdf=True):
+        """
+        Store the runner to an HDF structure.
+        Currently only output_nu and output_energy are stored.
+
+        Parameters
+        ----------
+        path_or_buf:
+            Path or buffer to the HDF store
+        path:
+            Path inside the HDF store to store the plasma
+        close_hdf : bool
+            If  True close the HDFStore [default=True]
+        filter: ~Filter
+            Only properties returned by the filter will be stored
+        Returns
+        -------
+            : None
+
+        """
+        try:
+            hdf_store = pd.HDFStore(path_or_buf)
+        except TypeError:
+            hdf_store = path_or_buf
+
+        properties = ['output_nu', 'output_energy']
```

Comment:
okay the properties that we want to store are `output_nu`, `output_energy`, `nu_bar_estimator`, `j_estimator`